File: app/src/main/java/com/swapniln/glabbrtest/groupMsgActivity.java
package com.swapniln.glabbrtest;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.swapniln.glabbrtest.ui.groupmsg.GroupMsgFragment;

public class groupMsgActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.group_msg_activity);
        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.container, GroupMsgFragment.newInstance())
                    .commitNow();
        }
    }
}

File: app/src/main/java/com/swapniln/glabbrtest/GroupMsgActivity.kt
package com.swapniln.glabbrtest

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

import com.swapniln.glabbrtest.ui.groupmsg.GroupMsgFragment

class GroupMsgActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.group_msg_activity)
        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction()
                    .replace(R.id.container, GroupMsgFragment.newInstance())
                    .commitNow()
        }
    }
}
